src/main/java/com/WebController/FrontController.java

```diff
@@ -0,0 +1,48 @@
+package com.WebController;
+
+import com.Model.Patient;
+import com.Repository.EventRepository;
+import com.Repository.PatientRepository;
+import com.Model.Event;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class FrontController {
```

Comment:
don't think this is a good name, as FrontController has specific meaning in Spring. Why not EventController, or maybe EventAttendanceController?